rero_mef/marctojson/do_gnd_concepts.py

```diff
@@ -0,0 +1,354 @@
+
+    def trans_gnd_deleted(self):
+        """Transformation deleted leader 5.
+
+        $c: Redirect notification
+        $x: Redirect
+        $c: Deletion notification
+        $d: Deletion
+
+        https://www.dnb.de/EN/Professionell/Metadatendienste/Datenbezug/
+        GND_Aenderungsdienst/gndAenderungsdienst_node.html
+        """
+        if self.logger and self.verbose:
+            self.logger.info("Call Function", "trans_gnd_deleted")
+        if self.marc.leader[5] in ["c", "d", "x"]:
+            self.json_dict["deleted"] = datetime.now(timezone.utc).isoformat()
+
+    def trans_gnd_pid(self):
+        """Transformation pid from field 001."""
+        if self.logger and self.verbose:
+            self.logger.info("Call Function", "trans_gnd_pid")
+        if field_001 := self.marc.get_fields("001"):
+            self.json_dict["pid"] = field_001[0].data
+            self.json_dict["type"] = "bf:Topic"
+
+    def trans_gnd_identifier(self):
+        """Transformation identifier from field 024, 035, 750 $i (exakte) Aequivalenz."""
+        if self.logger and self.verbose:
+            self.logger.info("Call Function", "trans_gnd_identifier")
+        fields_024 = self.marc.get_fields("024")
+        for field_024 in fields_024:
+            subfield_0 = field_024.get("0")
+            if isinstance(subfield_0, list):
+                subfield_0 = subfield_0[0]
+            subfield_2 = field_024.get("2")
+            if isinstance(subfield_2, list):
+                subfield_2 = subfield_2[0]
+            if subfield_0 and subfield_2:
+                self.json_dict.setdefault("identifiedBy", []).append(
+                    {
+                        "source": subfield_2.upper(),
+                        "type": "uri",
+                        "value": subfield_0,
+                    }
+                )
+        equivalences = []
+        exacte = {}
+        for field_750 in self.marc.get_fields("750"):
+            for subfield_0 in field_750.get_subfields("0"):
+                source, id_ = get_source_and_id(subfield_0)
+                if source and source != "DE-101":
+                    subfield_i = field_750.get("i")
+                    if subfield_i == "Aequivalenz":
+                        equivalences.append(
+                            {
+                                "source": source,
+                                "type": "bf:Nbn",
+                                "value": id_,
+                            }
+                        )
+                    elif subfield_i == "exakte Aequivalenz":
+                        exacte = {
+                            "source": source,
+                            "type": "bf:Nbn",
+                            "value": id_,
+                        }
+        if exacte:
+            self.json_dict.setdefault("identifiedBy", []).append(exacte)
+        for equivalence in equivalences:
+            self.json_dict.setdefault("identifiedBy", []).append(equivalence)
```

Comment:
I don't exactly understand how can I differentiate in the JSON data an standard equivalence and an exact equivalence? Also, the data in 750 is and equivalence, not identifiedBy. See below

Reply:
Added `exactMatch`.